### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by aclevername**
### Description

Sister PR in pctl https://github.com/weaveworks/pctl/pull/83
### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md` (diagrams, usage, roadmap etc), and anything in `examples/`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,34 +151,9 @@ Illustration of how Profiles are added to the Catalog and how they can then be q
 
 ![](/docs/assets/catalog.jpg)
 
-### Profile Subscriptions
+### Profile Installation
 
-Illustration of a basic profile install (aka subscription), using `kubectl` and a hand-written `ProfileSubscription`:
```
Comment (Callisto13) [inline README.md:199]: y'know a less lazy person would have updated the diagrams

Comment (aclevername) [reply README.md:199]: its no longer relevant though :shrug:

Comment (aclevername) [reply README.md:199]: I'll update the diagrams in `pctl`, since this repo is no longer related to the installation

Comment (aclevername) [reply README.md:199]: https://github.com/weaveworks/pctl/pull/87

**Review by Callisto13 — APPROVED** [review]

**Pull request merged**